frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -78,484 +94,1220 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
+                                            <LemonInput
+                                                data-attr="feature-flag-variant-name"
+                                                className="ph-ignore-input"
+                                                placeholder="Description"
+                                            />
+                                        </Field>
+                                    </Col>
+                                    <Col span={3}>
+                                        <Field name="rollout_percentage">
+                                            {({ value, onChange }) => (
+                                                <InputNumber
+                                                    min={0}
+                                                    max={100}
+                                                    value={value}
+                                                    onChange={(changedValue) => {
+                                                        if (changedValue !== null && changedValue !== undefined) {
+                                                            const valueInt = parseInt(changedValue.toString())
+                                                            if (!isNaN(valueInt)) {
+                                                                onChange(valueInt)
+                                                            }
+                                                        }
+                                                    }}
+                                                    style={{
+                                                        width: '100%',
+                                                        borderColor: areVariantRolloutsValid
+                                                            ? undefined
+                                                            : 'var(--danger)',
+                                                    }}
+                                                />
                                             )}
-                                        </div>
-                                        <Row align="middle">
-                                            <Tooltip title="Duplicate this condition set" placement="bottomLeft">
+                                        </Field>
+                                    </Col>
+                                    <Col span={2}>
+                                        <Row>
+                                            <LemonButton
+                                                icon={<IconCopy />}
+                                                status="primary-alt"
+                                                data-attr={`show-prop-duplicate-${index}`}
+                                                noPadding
+                                                onClick={() => {
+                                                    duplicateVariant(index)
```

Comment:
@neilkakkar @clarkus I'm not sure there's a lot of value in being able to duplicate variant keys, is it alright if I remove that? The user probably has to adjust the key, description, rollout % etc anyway even on duplication...

Reply:
We probably also want them to be intentional about choosing their variant keys

Reply:
I could go either way - it might be handy when you're duplicating and creating small variations on key names. We have an automatic rollout redistribution that can handle bulk updating distributions. I don't have a strong feeling either way, so I'd be fine removing it from scope unless we have a clear problem we're addressing.